src/lib/server/events/cache.ts

```diff
@@ -0,0 +1,5 @@
+import type { ClubEvent } from '$lib/public/events/event';
+import { Cache } from '$lib/server/cache/cache';
+
+// Make this false to disable server-side caching in development.
+export const allEvents = Cache.create<ClubEvent[]>(/* CACHING=*/ true);
```

Comment:
In the future, we may want to cache the events data on the server when "debug mode" is DISABLED. This allows the user to only have to change the value of DEBUG_FLAG_ENABLED in their `.env`.
```suggestion
import { DEBUG_FLAG_ENABLED } from '$lib/server/flags';

export const allEvents = Cache.create<ClubEvent[]>(/* CACHING=*/ !DEBUG_FLAG_ENABLED);
```